mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -73,3 +73,84 @@ table.mtp-table--review {
 .button-warning {
   @include button($red);
 }
+
+// TODO remove once new gov.uk styling implemented
+.moj-sub-navigation {
+  margin-bottom: 0.7rem;
+}
+
+
+.moj-sub-navigation__list {
+  font-size: 0; // Removes white space when using inline-block on child element.
+  box-shadow: inset 0 -1px 0 #b1b4b6;
```

Comment:
we should use `$grey-2` (or `$grey-3` if it looks a bit too dark) to pick up the old-style grey

Reply:
otherwise keep this if the constants both look poo